File: taotao-parent/taotao-portal-web/src/main/webapp/js/controller/contentController.js
//广告控制层（运营商后台）
app.controller("contentController",function ($scope,contentService) {

    //广告集合
    $scope.contentList = [];

    //根据分类ID查询广告列表
    $scope.findByCategoryId = function (categoryId) {
        contentService.findByCategoryId(categoryId).success(
            function (response) {
                $scope.contentList[categoryId] = response;
            }
        )
    };
});

File: taotao-parent/taotao-portal-web/src/main/java/com/taotao/portal/controller/ContentController.java
package com.taotao.portal.controller;

import com.alibaba.dubbo.config.annotation.Reference;
import com.taotao.content.service.ContentService;
import com.taotao.pojo.TbContent;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.List;

@RestController
@RequestMapping("/content")
public class ContentController {

    @Reference
    private ContentService contentService;

    /**
     * 根据广告分类ID查询广告列表
     */
    @RequestMapping("/findByCategoryId")
    public List<TbContent> findByCategoryId(Long categoryId){
        return contentService.findByCategoryId(categoryId);
    }
}
